terraform/modules/memcached/main.tf

```diff
@@ -1,31 +1,48 @@
-
 ##########################################################
 # Caches
 #
 # Memcached & Redis
 ##########################################################
+module "globals" {
+  source = "../globals"
+}
+
 resource "aws_elasticache_subnet_group" "ec" {
   name       = "tf-test-cache-subnet"
   subnet_ids = var.private_app_subnet_ids
 }
 
 resource "aws_elasticache_cluster" "memcached" {
-  cluster_id           = "memcached-spree"
+  cluster_id           = "scale-eu2-ec-spree-memcached"
   engine               = "memcached"
   node_type            = "cache.t3.medium"
   num_cache_nodes      = 2
   parameter_group_name = "default.memcached1.5"
   security_group_ids   = var.security_group_memcached_ids
   subnet_group_name    = aws_elasticache_subnet_group.ec.name
+
+  tags = {
```

Comment:
Could we set the repeating tags in a global map then use `merge` to combine with the `AppType` - save a bit of repetition / boilerplate? Something like:

```
tags = merge(module.globals.project_resource_tags, {AppType = "MEMCACHED"})
```

Reply:
Very nice idea - got it working - just about to push a commit with it in 👍

Reply:
Pushed